Routers/Routers/CreateGraph.cs

```diff
@@ -0,0 +1,90 @@
+/// <summary>
+/// class wich creates graph (topology) of input expression
+/// </summary>
+public class CreateGraph
+{
+    /// <summary>
+    /// method wich gets graph from input expression
+    /// </summary>
+    /// <param name="path">filepath</param>
+    /// <returns></returns>
+    /// <exception cref="NoSuchpathException">throw exseption if file doesnt exist </exception>
+    public Graphe GetGraph(string path)
+    {
+        if (!(File.Exists(path)))
+        {
+            throw new NoSuchpathException("No such file");
```

Comment:
```suggestion
            throw new NoSuchPathException("No such file");
```
ты тут поменял название исключения, а само исключение не поменял, поэтому ошибка компиляции